includes/discovery/storage/datadomain.inc.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/*
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2018 ACL
+ * @author     Abel Laura <abel.laura@gmail.com>
+*/
+
+if ($device['os'] == 'datadomain') {
```

Comment:
You can remove this, shouldn't be needed.

Reply:
I'll take of that today. Thanks.